furusiya/model/components/ai/monster.py

```diff
@@ -46,6 +46,11 @@ def _take_turn(self):
             self.owner.char = self.owner.name[0]
 
 
+class FrozenMonster(StunnedMonster):
```

Comment:
Is it possible to write tests for this? Somehow, can we test that the AI thaws out after a few turns?

Reply:
It's basically an empty subclass of StunnedMonster. I can't see any new tests being written, other than perhaps running the StunnedMonster's tests on this.

Reply:
Just realized AIs don't have tests. Oh well...

Reply:
Boyscout rule: leave it cleaner than you found it.

Next time, iA